exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/json/StoredConfigurationGsonSerializerTest.java

```diff
@@ -51,21 +51,29 @@ void nullPointerException() {
   }
 
   private StoredConfiguration createConfiguration() {
-    return StoredConfiguration.create(
-        HashCode.fromString("11"),
-        1,
-        singletonList(
-            ValidatorKey.create(HashCode.fromString("22"), HashCode.fromString("33"))
-        ),
-        ConsensusConfiguration.create(
-            1,
-            2,
-            3,
-            4,
-            5,
-            6,
-            7,
-            8)
-    );
+    return StoredConfiguration.builder()
+        .previousCfgHash(HashCode.fromString("11"))
+        .actualFrom(1)
+        .validatorKeys(
+            singletonList(
+                ValidatorKey.builder()
+                    .consensusKey(HashCode.fromString("22"))
```

Comment:
For a validator that might be an overkill, don't you think? Here, however, it improves the readability because we inline the constants, not named variables.

Reply:
Non essential as for me.